docs/drivers/aws.md

```diff
@@ -26,9 +26,9 @@ One way to configure credentials is to use the standard credential file for Amaz
     aws_access_key_id = AKID1234567890
     aws_secret_access_key = MY-SECRET-KEY
 
-You can learn more about the credentials file from this [blog post](http://blogs.aws.amazon.com/security/post/Tx3D6U6WSFGOK2H/A-New-and-Standardized-Way-to-Manage-Credentials-in-the-AWS-SDKs).
+On Mac OS or various flavors of Linux you can install the AWS Command Line Interface (aws cli) in the terminal and use the `aws configure` command which guide you through the creation of the credentials file.  For more information about getting started, visit the [AWS CLI documentation](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-quick-configuration).
```

Comment:
per @nathanleclaire's comment, I think it is actually nice to have a mention of the ability to use `aws configure` here.